cypress/integration/plp.test.js

```diff
@@ -20,26 +20,26 @@ describe('Search page Filters and Sorting options', () => {
 
     // Apply filters
     cy.getById('open-filter-button')
-      .click()
+      .click({ force: true })
       .getById('filter-accordion-button')
       .first()
-      .click()
+      .click({ force: true })
       .getById('filter-accordion-panel-checkbox')
       .should('exist')
       .first()
-      .click()
+      .click({ force: true })
       .getById('filter-modal-button-apply')
-      .click()
+      .click({ force: true })
 
     // Check applied filters
     cy.getById('open-filter-button')
-      .click()
+      .click({ force: true })
       .getById('filter-accordion-button')
       .first()
-      .click()
+      .click({ force: true })
       .getById('filter-accordion-panel-checkbox')
       .first()
-      .click()
+      .click({ force: true })
```

Comment:
Why did it need to have those forced clicks here?

Reply:
Yeah, @hellofanny, try removing these added `force: true` because I think they aren't needed anymore.